src/ApolloClient.re

```diff
@@ -1,31 +1,45 @@
 open ReasonApolloTypes;
 
 type variableTypeToBeDefined;
+
 type queryObj = {. "query": queryString, "variables": variableTypeToBeDefined};
-type generatedApolloClient = {. "query": [@bs.meth] (queryObj => string)};
-type clientOptions = {. "cache": unit, "link": unit};
-type linkOptions = {. "uri": string};
 
-[@bs.module "apollo-client"] [@bs.new] external apolloClient : clientOptions => generatedApolloClient = "ApolloClient";
+type generatedApolloClient = {. "query": [@bs.meth] (queryObj => string)};
 
-[@bs.module "apollo-link-http"] [@bs.new] external httpLink : linkOptions => 'a = "HttpLink";
+type linkOptions = {. "uri": string};
 
-[@bs.module "apollo-cache-inmemory"] [@bs.new] external inMemoryCache : unit => 'a = "InMemoryCache";
+[@bs.obj]
+external clientOptions :
```

Comment:
Usually this is named `makeXXX`, because you are "making" an object.

This is what I've seen so far at least 😇